### PR TITLE
Update Dockerfile and maven stuff to use java 11

### PR DESCRIPTION
**Pull request by AlvaroVega**
Upgrade docker and dependencies to build cygnus using java 11

Unfortunately there is a dependence which has to be updated
hadoop 1.2.1 and hive-jdbc are using java7 stuff which is not available in java11

### CODE REVIEW

**Comment by AlvaroVega** [comment]
@fgalan is there any way to tests this upgrade which implies upgrade hadoop ?

**Comment by AlvaroVega** [comment]
Current supported HDFS deployments are based on 
- AMBARI 2.7.3
- HDP 3.1.0  (which HIVE and hadoop versions according with https://docs.cloudera.com/HDPDocuments/HDP3/HDP-3.1.0/release-notes/content/comp_versions.html are: 
- Apache Hadoop 3.1.1
- Apache HBase 2.0.2
- Apache Hive 3.1.0

which are compatible with proposed upgrade:
- Upgrade hadoop-core from 1.2.1 to hadoop-client 2.2.0
- Upgrade hive-jdbc from 2.3.4 to 3.1.3

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -42,10 +42,7 @@ jobs:
       matrix:
         java-version:
           - 8
```
Comment (fgalan) [inline .github/workflows/ci.yml:44]:
```suggestion
```

File: CHANGES_NEXT_RELEASE [hunk 2]
```diff
@@ -1,3 +1,13 @@
+[cygnus-common][cygnus-ngsi] Upgrade Java version from 1.8 to 1.11 in Dockerfile
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE:1]:
```suggestion
[cygnus-common][cygnus-ngsi][cygnus-ngsi-ld] Upgrade Java version from 1.8 to 1.11 in Dockerfile
```

File: docker/cygnus-ngsi/Dockerfile [hunk 3]
```diff
@@ -340,7 +333,7 @@ RUN \
     find /usr/share/locale -mindepth 1 -maxdepth 1 ! -name 'en_US' ! -name 'locale.alias' | xargs -r rm -r && rm -f /var/log/*log && \
     bash -c 'localedef --list-archive | grep -v -e "en_US" | xargs localedef --delete-from-archive' && \
     /bin/cp -f /usr/lib/locale/locale-archive /usr/lib/locale/locale-archive.tmpl && \
-    #build-locale-archive && find ${CYGNUS_HOME} -name '.[^.]*' 2>/dev/null | xargs -r rm -rf && \
+    #build-locale-archive && find ${CYGNUS_HOME} -name '.[^.]*' 2>/dev/null | xa1rgs -r rm -rf && \
```
Comment (fgalan) [inline docker/cygnus-ngsi/Dockerfile:336]: xa1rgs is a typo?

**Review by fgalan — APPROVED** [review]
LGTM

File: docker/cygnus-ngsi/Dockerfile (hunk 3 above)
Comment (AlvaroVega) [reply docker/cygnus-ngsi/Dockerfile]: Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/2225/commits/b404fe4ba92b76dd041a709ac45a8a622bf309c6

**Comment by AlvaroVega** [comment]
related https://github.com/telefonicaid/fiware-cygnus/pull/2229

**Pull request merged**